src/components/AddNote.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/prop-types */
+import React, {useState} from "react";
+
+const AddNote = ({handleAddNote}) => {
+  const [noteText, setNoteText] = useState("");
+  const handleChange = (textAreaInput) => {
+    setNoteText(textAreaInput.target.value);
+  }
+
+  const handleSaveClick = (e) => {
```

Comment:
It doesn't appear that e is used in this function

Reply:
Done.